crates/sui-graphql-rpc/src/types/transaction_block_kind/end_of_epoch.rs

```diff
@@ -204,6 +206,7 @@ impl AuthenticatorStateExpireTransaction {
     }
 }
 
+/// System transaction corresponding to an expired JWK.
```

Comment:
feel like this comment is misplaced

Reply:
+1, I think this should just be deleted.

Reply:
(Still relevant).